release/changes.txt

```diff
@@ -8660,6 +8660,7 @@ Bug fix release:
 
 - Fixed Bad Segmentation of some files,
   as documented in: http://groups.yahoo.com/group/OmegaT/message/672
+  (https://sourceforge.net/p/omegat/mailman/message/36811754/)
```

Comment:
IMHO, yahoo URL can be removed because old URL had already been unreachable.

Reply:
I preferred to leave it for historical purpose, but you're right.

Reply:
Presumably the original link was to a threaded view. The equivalent view now would be https://sourceforge.net/p/omegat/mailman/omegat-users/thread/cflh3i%2Bkbe1%40.../#msg36811754

Reply:
>  for historical purpose,

IMHO, historical URL can be seen in git history when necessary.